test/.nycrc-report

```diff
@@ -0,0 +1,7 @@
+{
+  "reporter": [
+    "lcov",
+    "text",
+    "text-summary"
+  ]
+}
```

Comment:
Similarly, `"text-summary"` reporter does _not_ work when passed as a CLI argument (⁉️) so doing it in an extra file for now...